packages/validation/src/__test__/isDate.spec.js

```diff
@@ -0,0 +1,16 @@
+import isDate from '../isDate';
+
+describe('isDate', () => {
+    it('should return true', () => {
+        const values = [new Date(), '2018-12-19', '2018/04/04', '2018/04/04 12:00:00 am'];
```

Comment:
I don't know if we want to return true for string formatted like a valid date, isDate should return true only for Date instances